### PR TITLE
GH token description added to most common problems

### PR DESCRIPTION
**Pull request by AnetaCadova**

### CODE REVIEW

**Comment by syndesisqe** [comment]
Can one of the admins verify this patch?

File: README.md [hunk 1]
```diff
@@ -387,6 +387,8 @@ oc delete secret test-lock
 *  If tests failed at ***java.lang.IllegalArgumentException: bound must be positive***, 
 just add -Dsyndesis.config.openshift.namespace.lock=true parameter to the command.
 
+*  When you start minishift, you might receive a 403 forbidden status from GitHub if your request exceeds the rate limit for your IP address. In this case, the command will fail and you will receive an error message. You need to create a personal API token from your GitHub account at https://github.com/settings/tokens. After you generate the API token, you need to set the MINISHIFT_GITHUB_API_TOKEN environment variable by running: export MINISHIFT_GITHUB_API_TOKEN=<token_ID> .
```
Comment (asmigala) [inline README.md]: Please enclose the `export MINISHIFT_GITHUB_API_TOKEN=<token_ID>` in backticks to get nice formatting

```
`MINISHIFT_GITHUB_API_TOKEN=<token_ID>`
```

File: README.md [hunk 2]
```diff
@@ -350,7 +350,7 @@ You can use various parameters. From the previous command, parameter:
 * *syndesis.config.openshift.namespace.cleanup.after* - cleanup namespace after the tests (it can be useful to set this to false during debugging phase)
 
 You can use profile `-P deploy` that sets all 3 parameters to lock the namespace, clean the namespace and don't clean the namespace after tests.
-
+
```
[the remark this answers is not in the document]
Comment (asmigala) [reply README.md]: I meant the space here ^^, theres an extra ` ` at the start of the line. The space below is ok.

File: README.md [hunk 3]
```diff
@@ -386,6 +386,7 @@ oc delete secret test-lock
 
 *  If tests failed at ***java.lang.IllegalArgumentException: bound must be positive***, 
 just add -Dsyndesis.config.openshift.namespace.lock=true parameter to the command.
+*  When you start minishift, you might receive a 403 forbidden status from GitHub if your request exceeds the rate limit for your IP address. In this case, the command will fail and you will receive an error message. You need to create a personal API token from your GitHub account at https://github.com/settings/tokens. After you generate the API token, you need to set the MINISHIFT_GITHUB_API_TOKEN environment variable by running: export `MINISHIFT_GITHUB_API_TOKEN=<token_ID>` .
```
Comment (asmigala) [inline README.md]: My bad here, please also add the `export` in the ticks (my mistake when copy-pasting)

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**